contracts/src/LightClient.sol

```diff
@@ -37,6 +37,15 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     /// @notice upgrade event when the proxy updates the implementation it's pointing to
     event Upgrade(address implementation);
 
+    /// @notice the approved prover was updated
```

Comment:
Update the doc string to use "permissioned"?

Reply:
done